st2common/st2common/hooks.py

```diff
@@ -106,24 +104,5 @@ def _validate_token(request):
 
         token_in_headers = headers.get('X_Auth_Token', None)
```

Comment:
I didn't touch or change the key names - wsgi handling code in webob just does some weird key renaming during look-ups inside __getattr__.

I can test if X-Auth-Token also works and if does, use that to make it more clear and obvious.